manifests/server/replication.pp

```diff
@@ -0,0 +1,41 @@
+#
+class mysql::server::replication {
+  $master_slave  = $mysql::server::master_slave
+  $server_id     = $mysql::server::rep_server_id
+
+  if $mysql::server::replication_enable {
```

Comment:
Please make the inclusion of the replication class optional in the `mysql::server`, instead of protecting the contents. This makes it easier to compose manually, in those ugly edge cases.